src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -65,7 +74,42 @@ export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
         name={conversation.to.name}
         initials={conversation.to.initials}
       />
-      <Separator />
+      {item && (
+        <>
+          <Separator />
+          <Flex flexDirection="column" p={2}>
+            <Sans mb={2} size="3" weight="medium">
+              {item.__typename}
+            </Sans>
+            <Flex>
+              <a href={item.href}>
```

Comment:
Curious to know if there are differences between using `<a/>` or `<Link/>` here?

Reply:
This actually needs to be updated to `RouterLink` to ensure client side navigation works correctly. Ultimately, no, no real difference. `Link` from palette has some style properties that isn't really necessary for an image though.